tests/_helpers.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from unittest import mock
```

Comment:
The tests fail for Python 2.7 due to this import.

```suggestion
import mock
```

The system tests for Python 2.7 still fail with this fix. You can run the system tests locally [using nox](https://github.com/googleapis/python-spanner/blob/master/CONTRIBUTING.rst#using-nox) to help with debugging.

Reply:
whoops, should be good now (all tests pass locally) - thought I could get away with not installing 2.7 :P